snuba/querylog/query_metadata.py

```diff
@@ -15,6 +22,107 @@ class QueryStatus(Enum):
     TIMEOUT = "timeout"
 
 
+QUERY_ERROR_MAPPINGS = {
```

Comment:
Nit: Makes it easier to understand the purpose of these mappings
```suggestion
CLICKHOUSE_ERROR_TO_SNUBA_ERROR_MAPPINGS = {
```